src/fluree/server/consensus/events.clj

```diff
@@ -24,13 +24,16 @@
   (map? x))
 
 (defn with-txn
-  [evt txn]
-  (cond (txn-address? txn)
-        (assoc evt :txn-address txn)
+  [evt txn {:keys [format] :as _opts}]
```

Comment:
I don't think we should be pushing opts further into the call stack. If anything, we should be pushing them up further towards the api boundary.

I think this should be two different functions, one for turtle and one for json. One could call the other to avoid repeated code.